core/store/src/flat/storage.rs

```diff
@@ -517,6 +517,10 @@ mod tests {
             *self.height_to_hashes.get(&height).unwrap()
         }
 
+        fn get_block(&self, height: BlockHeight) -> BlockInfo {
```

Comment:
Where do we call it?

Reply:
in multiple places when creating `FlatStateDelta` for tests

Reply:
Oh, I see.